src/main/scala/io/citrine/lolo/transformers/standardizer/Standardizer.scala

```diff
@@ -0,0 +1,64 @@
+package io.citrine.lolo.transformers.standardizer
+
+import io.citrine.lolo.Learner
+import io.citrine.random.Random
+
+trait Standardizer[T] extends Learner[T] {
```

Comment:
The `Standardizer[T]` and associated interfaces don't offer much in terms of code; I didn't see much between regression and classification that could be generically moved to the interface. But I still opted to implement a trait just to unify the inheritance chain between regression and classification standardizers.